deployment/helm/synopsys-alert/README.md

```diff
@@ -130,3 +190,237 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 ```console
 $ helm install . --name <name> --namespace <namespace> --set enableStandalone=true
 ```
+
+### Persistent Storage
+The section will describe the changes needed to configure persistent storage.  
+**Please note:** By default the 'values.yaml' file has persistent storage disabled in order for Alert to work out of the box.
```

Comment:
It's actually enabled right now `enablePersistentStorage: true` by default

Reply:
Yes I would like it to be false so that Alert can start up right out of the box.  The second file changed in this PR sets the value to false.